styles.css

```diff
@@ -84,14 +89,11 @@ body{
     background: rgba(var(--rgba-light-theme-hover));
 }
 
-.header__btn--download {
-}
+.header__btn--download {}
 
-.header__btn--print {
-}
+.header__btn--print {}
 
-.header__btn--save {
-}
+.header__btn--save {}
```

Comment:
can be removed if not in use

Reply:
No, that's a placeholder for future use.